ember_debug/models/profile-manager.js

```diff
@@ -80,6 +163,10 @@ export default class ProfileManager {
     }
   }
 
+  teardown() {
+    this.stylesheet.remove();
```

Comment:
likely we should remove all "higlight" nodes too here (if exists)

Reply:
Added highlightComponents to track the active highlights.